td/gsheet/td_gsheet.dig

```diff
@@ -5,9 +5,10 @@ timezone: UTC
   database: sample_datasets
   result_connection: your_connections_name
   result_settings:
+    spreadsheet_folder: your_folder_id
     spreadsheet_title: destination_spreadsheet_name
     sheet_title: destination_worksheet_name
-    mode: APPEND
+    mode: replace
```

Comment:
`range` mode in this dig file has been meaningless as far as using [`append`](https://support.treasuredata.com/hc/en-us/articles/360009671913-Google-Sheets-Export#Append) mode, thus I change the option value to another one.